<?xml version="1.0" encoding="utf-8"?><messages><msg code="0" type="2">Start Time: vineri, 4 noiembrie 2022 11:16:07</msg><msg code="0" type="0">comune_PuncteInTorinoFuel</msg><msg code="0" type="0">comune_PuncteInTorino</msg><msg code="0" type="0">comune_PuncteInTorinoViaNizzaFuel</msg><msg code="0" type="0">comune_PuncteInTorinoViaNizza</msg><msg code="0" type="0">comune_PuncteInRiminiFuel</msg><msg code="0" type="0">comune_PuncteInTrento</msg><msg code="0" type="0">points</msg><msg code="0" type="0">comune_DrumuriInTorinoViaNizzaFuel</msg><msg code="0" type="0">comune_DrumuriInTorinoViaNizza</msg><msg code="0" type="0">comune_DrumuriInTorinoFuel</msg><msg code="0" type="0">comune_DrumuriInTorino</msg><msg code="0" type="0">roads</msg><msg code="0" type="0">comune</msg><msg code="0" type="0">World Topographic Map</msg><msg code="0" type="0">World Hillshade</msg><msg code="0" type="0">comune_DrumuriInTorinoViaNizza</msg><msg code="-2147467259" type="100">Traceback (most recent call last):
  File "D:\SCOALA\Master Semestrul 1\M3 - BD Spatiale\Seminar\Proiect\Proiect_BDS\Simbologie.py", line 124, in &lt;module&gt;
    ScriptTool(in_pct_sel=sel_loc)
  File "D:\SCOALA\Master Semestrul 1\M3 - BD Spatiale\Seminar\Proiect\Proiect_BDS\Simbologie.py", line 110, in ScriptTool
    itm.symbol.applySymbolFromGallery("Circle 1")
  File "C:\Program Files\ArcGIS\Pro\Resources\ArcPy\arcpy\_symbology.py", line 205, in applySymbolFromGallery
    self._arc_object.applySymbolFromGallery(*gp_fixargs((wildcard, index), True)))
TypeError: index out of range
</msg><msg code="0" type="0">Failed script (null)...</msg><msg code="-2147467259" type="100">Failed to execute (SimbologiePuncteComuna).</msg><msg code="0" type="3">Failed at vineri, 4 noiembrie 2022 11:16:07 (Elapsed Time: 0,24 seconds)</msg></messages>